podcasts/GeneralSettingsViewController.swift

```diff
@@ -351,23 +366,28 @@ class GeneralSettingsViewController: UIViewController, UITableViewDelegate, UITa
     }
 
     func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
-        if section == 1 {
+        guard let sectionItems = tableData[safe: section]?.last else { return nil }
```

Comment:
Thats even better then switching on sectionItems.last. Nitpick, but this could be just sectionItem now.

Reply:
I improved it in 5deebe969906b873e8a1cd95111f075a79b5ed6b

I also removed the `guard` since it's not needed